transformation_test/testdata/logging_processor-redis/output_fluentbit.yaml

```diff
@@ -0,0 +1,335 @@
+- entries:
```

Comment:
There are some test failures regarding this `logging_processor-redis` transformation test. This is probably because there is a mis-match with the expected output. Executing `make transformation_test_update` may solve the issue.